src/main/java/racingcar/domain/car/CarName.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain.car;
+
+import java.util.List;
+
+public class CarName {
+
+    private static final int MAX_NAME_LENGTH = 5;
+
+    private final String name;
+
+    public CarName(String name) {
+        this.name = name;
+    }
```

Comment:
정적 팩토리 메서드로 인스턴스를 생성하시는 것으로 보이는데
다른 클래스는 제한자가 잘 설정되어있지만
여기는 생성자 생성을 허용하는 하셨네용
실수인것 같아 알려드립니다!
```suggestion
private CarName(String name) {
        this.name = name;
    }
```

Reply:
으아악 ㅋㅋㅋ 실수 맞습니다! 항상 이런 부분을 주의하며 정적 팩토리 메서드를 쓰는데, 놓치고 말았네요.. 꼼꼼히 봐주셔서 감사합니다 🤠